csv_export.php

```diff
@@ -45,6 +45,11 @@
 require_api( 'helper_api.php' );
 require_api( 'print_api.php' );
 
+if( !access_has_project_level( config_get( 'export_issues_threshold' ) ) ) {
```

Comment:
I agree. 
I randomly look in file "account_prof_update.php" and see that access denied test was done before auth_ensure_user_authenticated(), so I did the same. Bad luck ?

	if( !config_get( 'enable_profiles' ) ) {
		trigger_error( ERROR_ACCESS_DENIED, ERROR );
	}

	form_security_validate( 'profile_update' );

	auth_ensure_user_authenticated();